src/docfx/restore/RestoreGit.cs

```diff
@@ -147,7 +147,10 @@ void AddWorkTrees()
                                             // https://stackoverflow.com/questions/24265481/after-directory-delete-the-directory-exists-returning-true-sometimes
                                             Directory.Delete(workTreePath, true);
                                             while (Directory.Exists(workTreePath))
+                                            {
+                                                Log.Write($"Wait for the {workTreePath} to be deleted");
                                                 Thread.Sleep(1);
```

Comment:
this is going to generate a zillion logs, consider writing a log every 30 seconds

Reply:
also could just use `Console.WriteLine` to show this message when `--verbose` is not specified

Reply:
it's not going to generate a zillion logs because the directly deleting delay is really short. And i want it to be showed when --verbose for debugging only.